guide/general/soulbind.md

```diff
@@ -114,4 +114,4 @@ Dreamweaver while it gives you an extra potency conduit.
 
 Korayn for mythic+ and raids with waves of adds.
 
-Niya for single target.
\ No newline at end of file
+Niya for single target.
```

Comment:
Any point talking about Dreamweaver?

Reply:
Dreamweaver is the more common ST build. If anything, Niya is the one that warrants less discussion.

Reply:
The soulbind page does no longer talk about any soulbind in specifics. It's just showing simulated values. Feel free to once more add specific content in a separate pull request